centreon-open-tickets/widgets/open-tickets/src/index.php

```diff
@@ -350,7 +350,7 @@
             'type' => PDO::PARAM_INT
         ];
     }
-    $instanceIdCondition = ' i.instance_id IN (' . $queryPoller . ')';
+    $instanceIdCondition = ' h.instance_id IN (' . $queryPoller . ')';
```

Comment:
I don't understand the fix because the 2 fields seems absolutely identical to me.
cf. line `175`
```
FROM hosts h JOIN instances i ON h.instance_id=i.instance_id
```

Reply:
I fixed it by deleting in my last commit